middleware/cloudinary.js

```diff
@@ -3,9 +3,9 @@ const cloudinary = require("cloudinary").v2;
 require("dotenv").config({ path: "./config/.env" });
 
 cloudinary.config({
-  cloud_name: process.env.CLOUD_NAME,
-  api_key: process.env.API_KEY,
-  api_secret: process.env.API_SECRET,
+    cloud_name: 'deija',
+    api_key: '551383875539282',
+    api_secret: 'LCiG2QBsgCHRpe6V1w5tcB50VXU'
```

Comment:
you can revert these to 
```
  cloud_name: process.env.CLOUD_NAME,
  api_key: process.env.API_KEY,
  api_secret: process.env.API_SECRET,
```

and then store `CLOUD_NAME`, `API_KEY`, and `API_SECRET` as environment variables.